docs/reference/environment.rst

```diff
@@ -26,6 +26,93 @@ variants.
 Supported variables
 -------------------
 
+EDGEDB_DOCKER_ABORT_CODE
```

Comment:
`edgedb-server` doesn't know anything about the `EDGEDB_DOCKER_*` variables. These are only used by the docker image. At least some of these are already documented here https://github.com/edgedb/edgedb/blob/master/docs/guides/deployment/docker.rst Maybe it makes sense to keep them separate?

Reply:
Actually there are others that the server doesn't know about too like `EDGEDB_SERVER_BOOTSTRAP_SCRIPT_FILE` that don't start with `EDGEDB_DOCKER_*`.

Reply:
> edgedb-server doesn't know anything about the EDGEDB_DOCKER_* variables. These are only used by the docker image. Maybe it makes sense to keep them separate?

Would you keep them on this page but divide them under separate headings?

> Actually there are others that the server doesn't know about too like EDGEDB_SERVER_BOOTSTRAP_SCRIPT_FILE that don't start with EDGEDB_DOCKER_*.

Is this a Docker env var that just didn't get the prefix, or is it something different?

Reply:
> Would you keep them on this page but divide them under separate headings?

The docker environment variables seem to cohere more with the docker deployment instructions rather than with the server environment variables to me. Readers only care about them when deploying with docker. But this may be subjective :shrug:.

> Is this a Docker env var that just didn't get the prefix, or is it something different?

Yea. I _think_ this is an OG docker var that was created before we had this naming convention.

Reply:
This change was originally inspired by [a Discord thread](https://discord.com/channels/841451783728529451/1058029984486273108) in which a user was expecting to find Docker env vars in this file as well, so it seems like people are looking for them here. It seems reasonable to me, and I think we can do it without any harm as long as we separate them and make clear that those Docker variables are invisible to the server. I'll give it a shot and see what I can come up with…

Reply:
Sure, sounds good. Maybe the environment variable section in the docker deploy guide should just be a link to this page then?

Reply:
> Actually there are others that the server doesn't know about too…

Is there an easy way I could discover which variables the server is aware of and which it isn't? I'll try to sort them into sections.

Reply:
I grep the server repo and the edgedb-docker repos.